### ISSUE TITLE
NVDA fails to echo typed characters in Microsoft News application

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#16990]
In the Microsoft News app, also known as Microsoft Bing News, NVDA doesn't echo typed characters inside the Search edit box or the Comment edit field. Narrator and JAWS aren't affected.

### Steps to reproduce:
1. Open Microsoft News. Typing "News" in the Start Menu oftentimes highlights it.
2. Press Tab a few times and then followed by "e" to reach the app's Search edit control. Note that the Comment edit box for each news story can also be used for this.
3. Activate the Focus mode, by pressing Enter, to be able to type into the edit box, and start typing there.
### Actual behavior:
NVDA remains silent as each character is being typed.
### Expected behavior:
NVDA should echo each typed character, respecting the "Speak typed characters" setting.
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
Alpha 33342, but many previous alpha releases are also affected.
#### Windows version:
Windows 11 24H2 (OS Build 26100.1301)
#### Name and version of other software in use when reproducing the issue:
Microsoft BingNews version 4.55.62231.0
#### Other information about your system:
Note: other screen readers like JAWS and Narrator function as expected.
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Many previous alphas over the last 2 weeks or so.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tried.

**Comment by Adriani90** [comment]
Following errors happen in the log after enabling focus mode on the searchfield and typing:

```
DEBUGWARNING - NVDAObjects.IAccessible.IA2TextTextInfo._getLineOffsets (20:39:44.551) - MainThread (20956):
IAccessibleText::textAtOffset failed
Traceback (most recent call last):
  File "NVDAObjects\IAccessible\__init__.pyc", line 425, in _getLineOffsets
  File "comtypes\_memberspec.pyc", line 629, in __call__
_ctypes.COMError: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
IO - inputCore.logTimeSinceInput (20:39:44.578) - MainThread (20956):
0.334 sec since input
IO - speech.speech.speak (20:39:44.578) - MainThread (20956):
Speaking ['Suchbegriff eingeben', 'Eingabefeld', CancellableSpeech (still valid), 'Leer']
IO - inputCore.InputManager.executeGesture (20:39:45.390) - winInputHook (3620):
Input: kb(laptop):leftArrow
DEBUG - editableText.EditableText._hasCaretMoved (20:39:45.501) - MainThread (20956):
Caret didn't move before timeout. Elapsed: 0.10728 sec
IO - inputCore.logTimeSinceInput (20:39:45.503) - MainThread (20956):
0.113 sec since input
IO - speech.speech.speak (20:39:45.503) - MainThread (20956):
Speaking ['Leer']
IO - inputCore.InputManager.executeGesture (20:39:45.780) - winInputHook (3620):
Input: kb(laptop):rightArrow
DEBUG - editableText.EditableText._hasCaretMoved (20:39:45.890) - MainThread (20956):
Caret didn't move before timeout. Elapsed: 0.105985 sec
IO - inputCore.logTimeSinceInput (20:39:45.892) - MainThread (20956):
0.112 sec since input
IO - speech.speech.speak (20:39:45.892) - MainThread (20956):
Speaking ['Leer']
IO - inputCore.InputManager.executeGesture (20:39:46.875) - winInputHook (3620):
Input: kb(laptop):a
IO - inputCore.InputManager.executeGesture (20:39:47.071) - winInputHook (3620):
Input: kb(laptop):s
IO - inputCore.InputManager.executeGesture (20:39:47.258) - winInputHook (3620):
Input: kb(laptop):d
DEBUG - NVDAObjects.IAccessible.IAccessible._getIA2TargetsForRelationsOfType (20:39:48.490) - MainThread (20956):
Got 0 relations, given maxRelations: 1
```

So this issue is still reproducible with NVDA alpha-53511,1472d4b9 (2026.1.0.53511)

cc: @cary-rowen
